### ISSUE TITLE
[Bug] Contacts from my address book that have Twake, aren't displayed as Twake contacts

### ISSUE THREAD

**Issue by tprudentova** [linagora/twake-on-matrix#2007]
2.6.1, tested on Android, linagora.com, staging

1. Create a TWP account with phone number XXX 
2. Add this phone number to your device's address book 
3. Go to your Linagora account and synchronize contacts 
4. Find phone number XXX in the list of contacts 
5. The contact isn't displayed as a Twake user, you can't start a chat with them 

Expectation: when a contact from my address book has a Twake account, I see them in the list of contacts as a Twake user 

https://github.com/user-attachments/assets/8fa72ef6-8d6c-41a6-94f9-47b20ff46f42

**Comment by hoangdat** [comment]
yes, TWP was not have this function. Need to have it when contact synchronization complete
